### ISSUE TITLE
How do we localize the “Ask for help with this question” links?

### ISSUE THREAD

**Issue by zoul** [theodi/open-data-certificate#1333]
There’s a “Ask for help with this question” link next to each question, leading to the discussion forum in the web app. Where do we point the link in the localized version of the app? I think that the institutions filling in the questionnaire would be best troubleshooted by their local open data communities, so the link should lead to a local forum instead of the ODI one. What do you think?

**Comment by Stephen-Gates** [comment]
Agreed. I would like to customise the link to point to the ODI Queensland discussion forum to support the Australian localisation
